File: app/src/main/java/com/nimtego/plectrum/presentation/main/model/ArtistModel.java
package com.nimtego.plectrum.presentation.main.model;

import lombok.Builder;
import lombok.Data;
import lombok.ToString;

@Data
@Builder
@ToString
public class ArtistModel {
    private String artistName;
    private String artistArtwork;
    private String primaryGenreName;
    private String artistViewUrl;
    private String artistId;
}

File: app/src/main/java/com/nimtego/plectrum/presentation/mvp/model/music/ArtistModel.kt
package com.nimtego.plectrum.presentation.mvp.model.music

data class ArtistModel(
        val artistName: String,
        val artistArtwork: String,
        val primaryGenreName: String,
        val artistViewUrl: String,
        val artistId: String,
        val artistAlbums: List<AlbumModel> = emptyList())